docs/src/main/paradox/common/http-model.md

```diff
@@ -67,6 +67,19 @@ simplify creating requests for common cases. Also, to aid readability, there are
 named after HTTP methods to create a request with a given method and URI directly.
 @@@
 
+@@@ note { title='String representation' }
+
+There are certain environments where it is easy to inadvertently print, write or log entries built out of string representations of @apidoc[HttpRequest] instances. On the other hand, it is not uncommon for HTTP headers and entities to contain _Personal Identifying Information (PII)_ or _Sensitive Personal Information (SPI)_ . 
+
+For all this, if @apidoc[HttpRequest] `toString` method included body and/or headers, it would be easy to accidentally violate privacy laws such as _GDPR_. These fields are, therefore, omitted from the string representation. 
```

Comment:
I'm not sure we should be making such legal-ish statements in our documentation - perhaps leave it at "To avoid accidentally leaking such information into the logging these fields are omitted from the `toString` output."?

Reply:
> erhaps leave it at "To avoid accidentally leaking such information into the logging these fields are omitted from the toString output."

:100: Changing to that.

Reply:
Done:

```
To avoid accidentally leaking such information, these fields are omitted from @apidoc[HttpRequest] `toString` output. 
```